cmake/app/boilerplate.cmake

```diff
@@ -166,6 +166,13 @@ elseif(DEFINED ENV{ZEPHYR_EXTRA_MODULES})
   set(ZEPHYR_EXTRA_MODULES $ENV{ZEPHYR_EXTRA_MODULES})
 endif()
 
+# 'MODULE_EXT_ROOT' is a prioritized list of directories where module code may
```

Comment:
... where module glue code may be found....

Reply:
Done